docs/contract_metadata.md

```diff
@@ -0,0 +1,26 @@
+Contract Metadata
```

Comment:
Can you name it something different to avoid confusion with the "other" contract metadata? :)

Truffle uses the term "build artifact" for the same.